ex01.exs

```diff
@@ -28,8 +28,22 @@ defmodule Ex01 do
   """
   
   def counter(value \\ 0) do
+    receive do
+      {:next,from} ->
+	send from,{:next_is,value}
+    end
+    counter(value+1)
   end
 
+  def new_counter(value), do: spawn __MODULE__,:counter,[value]
+
+  def next_value(pid) do
+    send pid,{:next,self()}
+    receive do
```

Comment:
No deduction, but please be consistent with layout and spaces:

~~~ elixir
{:next_is, value} ->
  value
~~~